routing/routing_session.hpp

```diff
@@ -34,13 +35,34 @@ class RouteMatchingInfo;
 
 namespace routing
 {
-struct SpeedCameraRestriction
+struct SpeedCameraOnRoute
 {
-  size_t m_index;  // Index of a polyline point where camera is located.
-  uint8_t m_maxSpeedKmH;  // Maximum speed allowed by the camera.
+  SpeedCameraOnRoute() = default;
+  SpeedCameraOnRoute(double distFromBegin, uint8_t maxSpeedKmH)
+    : m_distFromBeginMeters(distFromBegin), m_maxSpeedKmH(maxSpeedKmH)
+  {}
+
+  // Needs for calculate distance for slow down speed before speed camera.
+  static double constexpr kAverageAccelerationOfBraking = -6.4697;  // Meters to square seconds.
+  static_assert(kAverageAccelerationOfBraking != 0, "");
+  static_assert(kAverageAccelerationOfBraking < 0, "AverageAccelerationOfBraking must be negative");
+
+  static double constexpr kDangerousZoneMeters = 450.0;  // Influence zone of speed camera.
+  static double constexpr kDistanceEpsilonMeters = 10.0;
+  static double constexpr kDistToReduceSpeedBeforeUnknownCameraM = 50.0;
```

Comment:
nit: other variables are Meters and this one is only M

Reply:
it is too long, so use `M` not `Meters`